Implement initial version (#2)
ioxiocom/multi-start: README.md

# multi-start

> 

Reviewer: You might want to add some badges, similar to how we have in other projcets.

This tool aims to help running multiple services inside a single docker container.

Sometimes you might want to have backend, frontend and nginx (or a combination of those)
inside a single container. This tool may help with:

- Prepare final Nginx configs using
  [parse-template](https://github.com/cocreators-ee/parse-template)
- Wait until backend and frontend start responding before running Nginx

> - Stop every process if one of them is shut down so a container stops gracefully

Reviewer: ```suggestion
- Stop every process if one of them exits so the whole container stops gracefully
```
